Editor/Scripts/interactivetutorials_dialog.py

```diff
@@ -152,6 +155,8 @@ def load_tutorial(self, index):
 
         tutorial_factory = self.tutorials[index]["tutorial"]
         self.current_tutorial = tutorial_factory()
+        
+        self.current_tutorial_num_steps = len(self.current_tutorial.get_steps())
```

Comment:
Need to subtract 1 for the 0 index.

```suggestion
        self.current_tutorial_num_steps = len(self.current_tutorial.get_steps()) - 1
```